lib/circuitbox/circuit_breaker.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
`frozen_string_literal` is only introduced since Ruby 2.3 if I remembered correctly. What if a client that's using some ruby version below that?